build.gradle

```diff
@@ -82,3 +82,4 @@ dependencies {
 tasks.named('test') { task ->
     useJUnitPlatform()
 }
+targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
are you sure need to include this? seems outside scope of bug fix. 

is this new standard for this sdk, I thought sdk was meant to go from 1.6+, maybe we've let that go, libs like 'org.threeten:threetenbp are in here due to 1.6 compat, whereas if 1.8 is used, it has that same date/time updates in jre packages.